hastexo/admin.py

```diff
@@ -3,7 +3,7 @@
 """
 from django import forms
 from django.contrib import admin
-from student.models import AnonymousUserId
+from common.djangoapps.student.models import AnonymousUserId
```

Comment:
So, just checking: do we know that this import also works on edx-platform Juniper? (I ask because the tests can't tell us, since we deliberately fake this out).

If it does not, I suggest we change this to:

```python
try:
    from common.djangoapps.student.models import AnonymousUserId
except ImportError:
    from student.models import AnonymousUserId